lib/Sema/SemaResolveInherit.cpp

```diff
@@ -353,6 +353,7 @@ void Sema::InjectInheritContract(std::vector<Decl *> &BaseDecl,
 
       BaseDecl.erase(ParentDeclIt);
     } else if (auto pDecl = dynamic_cast<VarDecl *>(*ParentDeclIt)) {
+      (void)!pDecl; // silence compiler warning
```

Comment:
Use `maybe_unused` to remove this warning.
https://en.cppreference.com/w/cpp/language/attributes/maybe_unused